Paintroid/src/main/java/org/catrobat/paintroid/presenter/MainActivityPresenter.java

```diff
@@ -431,8 +431,6 @@ public void onBackPressed() {
 			drawerLayoutViewHolder.closeDrawer(Gravity.END, true);
 		} else if (model.isFullscreen()) {
 			exitFullscreenClicked();
-		} else if (toolController.toolOptionsViewVisible()) {
-			toolController.hideToolOptionsView();
```

Comment:
Errors are all fixed, but back to 5) aren't that the lines of code for instantly setting the brush tool when pressing back? On develop branch the behavior is as desired: If option window is open and user presses back, the option window is closed. I would also prefer that brush tool option window gets closed when user touches the drawing surface but that's just a matter of taste again. Going to put it into PO review and let Wolfgang decide if changes are needed or not.